src/renderer/components/composer/ComposerMessageInput.tsx

```diff
@@ -92,6 +127,24 @@ export default class ComposerMessageInput extends React.Component<
   clearText() {
     this.setState({ text: '' })
   }
+  
+  startRecording() : any {
```

Comment:
does not return anything
```suggestion
  startRecording() {
```